typescript/environments.d.ts

```diff
@@ -40,13 +40,22 @@ export type FormsAppEnvironmentStyles = BaseFormsAppEnvironmentStyles & {
   }
 }
 
+export type FormsAppEnvironmentSendingAddress = {
+  emailAddress: string
+  emailName?: string
+  formsAppEnvironmentId: number
+  createdAt: string
+  updatedAt: string
```

Comment:
If we are storing this as on the environment, we dont need these props. I think storing as a seperate table is prob ideal though.

Reply:
@Zaxist are you planning on storing the environment sending address in a separate table?